app/domain/streams/handlers/single_item_handler.rb

```diff
@@ -16,6 +19,8 @@ def process
 private
 
   def find_old_edition(content_id, locale)
+    raise MissingLocaleError unless locale
```

Comment:
👍 I like the approach of guarding against unexpected data and deal with the errors in Sentry. It would be good to count how many content items that are not `redirect` or `gone(s)`don't have a locale. It the number is too big, we might have issues when repopulating the Editions dimension.

Reply:
> It would be good to count how many content items that are not `redirect` or `gone(s)`don't have a locale. It the number is too big, we might have issues when repopulating the Editions dimension.

I'm pretty sure this is 0. Everything in the Publishing API has a locale.